docs/source/examples.rst

```diff
@@ -1,15 +1,23 @@
 Examples
 #########
 
```

Comment:
Adding this here so I don't forget. Make sure to add the `preprocessing` module to the `api` in the docs. So you have to add the header and manually add functions but then also build the associated pages, per my instructions in the contribution guide.